src/dlr_tvm.cc

```diff
@@ -198,6 +198,12 @@ void TVMModel::GetOutput(int index, void* out) {
   get_output(index, &output_tensor);
 }
 
+const void* TVMModel::GetOutputPtr(int index) const {
+  tvm::runtime::NDArray output = tvm_graph_runtime_->GetOutput(index);
+  const DLTensor* tensor = output.operator->();
+  return tensor->data;
```

Comment:
Depending on the context, this pointer could be a regular CPU pointer or it may point to GPU memory. Should we leave it up to the customer to make sure they use it correctly? Or should we only allow this function for CPU context?

Reply:
Lets support CPU tensors only for now. We can investigate it and support other tensor types later.